### PR TITLE
discord-canary: 0.0.96 -> 0.0.97

### PR DESCRIPTION
**Pull request by tadeokondrak**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/discord/default.nix [hunk 1]
```diff
@@ -27,10 +27,10 @@ in {
     pname = "discord-canary";
     binaryName = "DiscordCanary";
     desktopName = "Discord Canary";
-    version = "0.0.96";
+    version = "0.0.97";
     src = fetchurl {
-      url = "https://dl-canary.discordapp.net/apps/linux/0.0.96/discord-canary-0.0.96.tar.gz";
-      sha256 = "1fxyh9v5xglwbgr5sidn0cv70qpzcd2q240wsv87k3nawhvfcwsp";
+      url = "https://dl-canary.discordapp.net/apps/linux/0.0.97/discord-canary-0.0.97.tar.gz";
```
Comment (jonringer) [inline pkgs/applications/networking/instant-messengers/discord/default.nix]: make bumping it easier
```suggestion
      url = "https://dl-canary.discordapp.net/apps/linux/${version}/discord-canary-${version}.tar.gz";
```

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
confirmed broken on master, fixes build
executable works

```
[3 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/70978
1 package were build:
discord-canary
```

**Comment by jonringer** [comment]
I didn't realize there was a script to update it ... nice :)

**Comment by tadeokondrak** [comment]
Note that when Discord is out of date it completely fails to launch:

**Comment by jonringer** [comment]
still builds, and works. Thanks for updating the update script! :)

**Pull request merged**